package.json

```diff
@@ -37,10 +37,11 @@
     "cookie-parser": "^1.3.5",
     "express": "^4.13.1",
     "express-session": "^1.11.3",
+    "jasmine-spec-reporter": "^4.2.1",
```

Comment:
Is this dependency actually needed? I'm fine with omitting it since we have generated the static test output